mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -1003,7 +1010,7 @@ def _input_generation(self, coords, patch_center, coord_max, attributes,
 
         return points
 
-    def _patch_points_sampling(self, points, sem_mask, replace=None):
+    def _patch_points_sampling(self, points, sem_mask):
```

Comment:
This function is only called inside this transform, so won't introduce BC-breaking.